### ISSUE TITLE
Issue with any version above 2.2.0

### ISSUE THREAD

**Issue by stan-ko** [johnkil/Android-RobotoTextView#50]
Hi!
I've got issues with the lib if using any version above 2.2.0, starting with 2.3.0. 
FOA the text in EditText looks like it has gravity top (2.2.0 - everything is ok) and if I tap the editbox the text vanishes like its textcolor becames white on white bg editText but the bigger issue gere - no keyboard appears so I'm unable to enter any text into editText. 
Also buttons starts ignoring gravity of its text, so the text label with v 2.2.0 is centered but if I switch to 2.3.0 text appears at left instead.
You could see the diff here - http://prntscr.com/akfos1
The same applies to latest 2.50 version.
I don't use appCompat.v7 in my app but olny support.v4 of 23.1.1 version. However adding appcompat.v7 lib to build.gradle does not change anything. EditText are locked from entering text and almost every TextView/EditText looks like it has gravity top. Every single layout appears broken and looks strange...

**Comment by johnkil** [comment]
Are you sure? I have a text is always in the middle, and the input is not locked. (RobotoTextView 2.5 & Android 6.0.1).

``` xml
<FrameLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="match_parent"
    android:layout_height="match_parent">

    <com.devspark.robototextview.widget.RobotoEditText
        android:id="@+id/edt_text"
        android:layout_width="match_parent"
        android:layout_height="100dp"/>

</FrameLayout>
```

**Comment by johnkil** [comment]
@stan-ko try to test on sample project.

**Comment by stan-ko** [comment]
Like I'd said I do not use AppCompat support lib in my project and to get issue I need only 1 thing to do - change the version of your lib in my build.gradle and nothing more. Here is my build.gradle, maybe you will see something conflicting setting you know:
```
buildscript {
    repositories {
        maven { url 'https://maven.fabric.io/public' }
    }

    dependencies {
        classpath 'io.fabric.tools:gradle:1.+'
    }

}
apply plugin: 'com.android.application'
apply plugin: 'io.fabric'

repositories {
    maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
    maven { url 'https://maven.fabric.io/public' }
}


android {

    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_7
        targetCompatibility JavaVersion.VERSION_1_7
    }
    compileSdkVersion 23
    buildToolsVersion '23.0.2'

    defaultConfig {

        minSdkVersion 10
        targetSdkVersion 16
        ndk {
            moduleName "Superpowered"
        }
        versionCode 15
        versionName "3.5 dev"
    }
    sourceSets.main {
        jniLibs.srcDir 'src/main/jniLibs'
        jni.srcDirs = [] //disable automatic ndk-build call
    }
    buildTypes {

        debug {
            buildConfigField "Boolean", "DEBUG_MODE", "true"
        }

        release {
            buildConfigField "boolean", "DEBUG_FLAG", "false"
            apply plugin: 'idea'
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
        }
    }
}

crashlytics {
    enableNdk true
    androidNdkOut 'src/main/jniLibs'
    androidNdkLibsOut 'src/main/jniLibs'
}

dependencies {
    // Crashlytics SDK Kit
    compile('com.crashlytics.sdk.android:crashlytics:2.5.2@aar') {
        transitive = true;
    }
    // Crashlytics NDK Kit
    compile('com.crashlytics.sdk.android:crashlytics-ndk:1.0.0@aar') {
        transitive = true
    }
    compile fileTree(dir: 'libs', include: ['*.jar'])

    // Google support Libraries
    compile 'com.android.support:support-v4:23.1.1'
    compile 'com.android.support:design:23.1.1'
    // Google f-g play services
    compile 'com.google.android.gms:play-services:4.2+'
    // Google GSON/JSON
    compile 'com.google.code.gson:gson:2.6.2'
    // EventBus
    compile 'de.greenrobot:eventbus:2.4.0'
    // OK HTTP + Retrofit
    compile 'com.squareup.okhttp:okhttp:2.0.0'
    compile 'com.squareup.okhttp:okhttp-urlconnection:2.0.0'
    compile 'com.squareup.retrofit:retrofit:1.9.0'
    // Camera Utils (available on github)
    compile project(':camera_utils')
    // cool cropper! based on 'com.edmodo:cropper:1.0.1'
    compile project(':cropper')
    // rounded/circle ImageView
    compile 'de.hdodenhof:circleimageview:1.2.2'
    // Images loading lib
    compile 'com.github.bumptech.glide:glide:3.7.0'
    // Roboto font TextViews
    compile 'com.github.johnkil.android-robototextview:robototextview:2.2.0'
    //Swipe layout
    compile 'com.daimajia.swipelayout:library:1.2.0@aar'
    // lib Phone Number formatter
    compile 'com.googlecode.libphonenumber:libphonenumber:7+'
    // StanKoUtils - a set of helper utils and tools
    compile 'com.github.stankoutils:app:1.3.3'
    // FAB
    compile 'com.melnykov:floatingactionbutton:1.3.0'

    //gif animation
    //compile 'pl.droidsonroids.gif:android-gif-drawable:1.1.11'
}
```
and here is a button which has gravity change issue:
```
        <com.devspark.robototextview.widget.RobotoButton
            android:id="@+id/btnLogin"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:background="@drawable/btn_common_blue_selector"
            android:padding="@dimen/padding_6"
            android:text="@string/login"
            android:textColor="@color/dark_gray"
            android:textSize="@dimen/text_size_button"
            app:textWeight="medium" />
```
nothing really special (wrapers are RelativeLayout and LinearLayout, no SwipeLayout or any other custom layout),
And the EditText:
```
            <com.devspark.robototextview.widget.RobotoEditText
                android:id="@+id/editEMail"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:background="@color/white"
                android:drawableLeft="@drawable/ico_mail"
                android:hint="@string/login_hint"
                android:inputType="textEmailAddress"
                android:paddingBottom="@dimen/editText_paddingTopBottom"
                android:paddingTop="@dimen/editText_paddingTopBottom"
                android:textSize="@dimen/text_size_edit_text" />
```
And there is nothing special here too.

Also I can't apply AppCompat since it breaks colors in my app if extending AppCompatActivity.

Also here is my styles:
```
<resources>

    <!--
        Base application theme, dependent on API level. This theme is replaced
        by AppBaseTheme from res/values-vXX/styles.xml on newer devices.
    -->
    <style name="AppBaseTheme" parent="android:Theme.Light">
        <!--
            Theme customizations available in newer API levels can go in
            res/values-vXX/styles.xml, while customizations related to
            backward-compatibility can go here.
        -->
    </style>

    <!-- Application theme. -->
    <style name="AppTheme" parent="android:Theme.Light.NoTitleBar">
        
        <!-- All customizations that are NOT specific to a particular API-level can go here. -->
        <item name="android:windowNoTitle">true</item>
        <item name="android:windowBackground">@color/main_blue</item> 
    </style>
```

**Comment by johnkil** [comment]
you don't need to extend AppCompatActivity, only change theme to Theme.AppCompat. This solves the problem.

**Comment by stan-ko** [comment]
You are right, its not about AppCompatActivity. Actually changing theme to Theme.AppCompat is the main problem to me,

**Comment by johnkil** [comment]
I'll try to rebuild the project without appcompat.

**Comment by johnkil** [comment]
https://oss.sonatype.org/content/repositories/snapshots/com/github/johnkil/android-robototextview/robototextview/2.5.1-SNAPSHOT/

**Comment by johnkil** [comment]
How to use in gradle:

```groovy
repositories {
    jcenter()
    maven {
        url "https://oss.sonatype.org/content/repositories/snapshots"
    }
}

dependencies {
    compile 'com.github.johnkil.android-robototextview:robototextview:2.5.1-SNAPSHOT'
}
```

**Comment by stan-ko** [comment]
Thanx, that 2.5.1-SNAPSHOT has no issues ) Would you include this fix in official 2.5.1 version or this will be only SNAPSHOT build?

**Comment by johnkil** [comment]
yes, in near future)
